src/reactapp/src/api/countries/fetchCountryList/modifier.js

```diff
@@ -1,8 +1,9 @@
 import _get from 'lodash.get';
 
 export default function fetchCountryListModifier(result) {
-  return _get(result, 'data.countries', []).map(country => ({
+  return _get(result, 'data.countries', []).map((country) => ({
     id: country.id,
     name: country.full_name_locale,
+    state_required: country.state_required,
```

Comment:
I prefer camelCasing here. That convention is followed in the repo in most of the places. So this is just suggestion